docs/conf.py

```diff
@@ -3,9 +3,6 @@
 import sys
 from datetime import date
 
-import recommonmark
```

Comment:
dont we need recommonmark to parse MD?

Reply:
Recommonmark it's still installed, but we don't need to do this custom setup anymore:

![image](https://user-images.githubusercontent.com/9107969/152113411-1422dae6-1bed-42d3-8ec3-99cf2d1c1f53.png)

Now, only need to list it as a extension:

![image](https://user-images.githubusercontent.com/9107969/152113465-4b85d781-f293-45dc-a1c0-f76f171712ed.png)